src/components/fields/SchemaField.js

```diff
@@ -113,7 +113,9 @@ function DefaultTemplate(props) {
     <div className={classNames}>
       {displayLabel && <Label label={label} required={required} id={id} />}
       {displayLabel && description ? description : null}
-      {children}
+      {React.Children.map(children, child => {
+        return React.cloneElement(child, { rawErrors: props.rawErrors });
+      })}
```

Comment:
This seems a bit heavy. Why is this necessary? Wouldn't it be better to pass the `rawErrors` prop when we constructed widgets?

Reply:
I've just reviewed the code again and I see that this can be passed through the SchemaFieldRender function. I'll update this.